packages/paste-core/components/form/stories/radio.stories.tsx

```diff
@@ -94,6 +94,44 @@ storiesOf('Forms|Radio', module)
       </Radio>
     );
   })
+  .add('Radios - Controlled', () => {
+    const [value, setValue] = React.useState('1');
+    const handleChange = (event: React.ChangeEvent<HTMLInputElement>): void => setValue(event.target.value);
+    return (
+      <>
```

Comment:
Nit: we don't technically allow this in our docs, as a radio must be part of a radio group, but I get why it's pulled out to show controlled state works on it's own

Reply:
Yeah I realize that this is a departure from the original design. I'm thinking we don't document this behavior _perse_ , but  it's helpful to showcase how it works in a controlled manner just in storybook for devs.

Reply:
Question; Are you able to understand how to use RadioGroup with Formik, given that RadioGroup is essentailly a requirement from our library for accessibility reasons?

Asked another way, by giving you an example that _doesn't_ use it, are we in danger of confusing people when we say "you must use the RadioGroup to wrap radios?

Reply:
I haven't really dug into Formik in a long time, but iirc Formik works best with simple controlled components.  The fact that RadioGroup is controlling Radios would add complexity to implementation.